indexer/script.js

```diff
@@ -53,7 +73,48 @@ export const config = {
   },
 };
 
-// This transform does nothing.
-export default function transform(block) {
-  return block;
+//This transformation filters events from the NFT Transfer event
+export default function transform({ events }) {
```

Comment:
Currently, things are set up to be interpreted on the backend. This logic should be carried out there : https://github.com/keep-starknet-strange/art-peace/blob/main/backend/routes/indexer.go

Reply:
Alright.